lib/membrane/rtp/ssrc_router.ex

```diff
@@ -15,9 +15,32 @@ defmodule Membrane.RTP.SSRCRouter do
 
   alias Membrane.{RTCP, RTP, RTCPEvent, SRTP}
 
+  require Membrane.TelemetryMetrics
+
   def_input_pad :input, caps: [RTCP, RTP], availability: :on_request, demand_mode: :auto
 
-  def_output_pad :output, caps: RTP, availability: :on_request, demand_mode: :auto
+  def_output_pad :output,
+    caps: RTP,
+    availability: :on_request,
+    demand_mode: :auto,
+    options: [
+      keyframe_detector: [
+        spec: function() | nil,
+        default: nil
+      ],
+      frame_detector: [
+        spec: function() | nil,
+        default: nil
+      ],
```

Comment:
I would encapsulate this in one function called e.g. `stats`.

Reply:
Maybe ssrc router could take these himself using `PayloadFormat.get`? Also, maybe create a behaviour and have a module containing `detect_keyframe` and `detect_frame` for each protocol?

Reply:
There are 2 issues with this approach:
1. Only VP8 format has both `frame_detector` and `keyframe_detector`, OPUS and H264 have only one of these
2. Currently, there is no dependency in `membrane_rtp_plugin` to `membrane_rtp_vp8_plugin` etc, so I would have to add them or pass this module in place, where I pass `frame_detector` and `keyframe_detector`, so I don't think if doing this would change a lot (assuming, that I don't want to add these dependencies)

Reply:
SSRC router could get the module the way Session Bin gets payloaders and depayloaders. That won't require RTP plugin to depend on rtp_vp8 etc, just like it doesn't depend on them now, still being able to get payloaders. Only the end-user would have to depend on them for this to work. If some codecs don't implement particular functions, they could either not have them in the said module and we'd have to check for them, or we could provide some default if possible.